src/sagas/tokens.js

```diff
@@ -127,7 +127,7 @@ function* fetchTokenHistory(action) {
     }
 
     const response = yield call(wallet.getTxHistory.bind(wallet), { token_id: tokenId });
-    const data = response.map((txHistory) => mapTokenHistory(txHistory, tokenId));
+    const data = response.map((item) => mapTokenHistory(item, tokenId));
```

Comment:
why?

Reply:
I did some tests and refactors in this method, the necessary change went to the lib and when i went to undo the changes I forgot this one.